### PR TITLE
OMF updates

### PR DESCRIPTION
**Pull request by afloresOSI**
This pull requests adds the steps from the OMF quick start guide to the OMF configuration section and adds more context.
Greg, the steps are the same as the quick start - if you could just look at the paragraphs of text in the topic to make sure they are accurate. I wasn't sure how much to talk about OMF with the PI System since that is a possible destination versus OMF with SDS/OCS.

### CODE REVIEW

File: V1/OMF/OMF_Messages_1-0.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+Message headers allow you to pass additional information with the message. The message header is where you specify the action for the message, such as CREATE. For a description of each of the headers, see the [OMF specification](http://omf-docs.osisoft.com). 
+
+The ``omfversion`` header must match the version of the OMF spec used to construct the message.
+Versions 1.0 and 1.1 of the spec are currently supported. 
+
+## Message Types
+
+OMF message types fall into three categories: Type, Container, and Data, which are described below. 
+Each message type creates a different type of data and contains keywords that define characteristics of the data. Most of the message types are used to create the structure of the data and to give it meaning. Data messages contain time-series data for which the PI System is known. The message types and the data they create are described in detail in this section. For details about the keywords supported by OCS, see the OCS topic, [Use OSIsoft Message Format with OSIsoft Cloud Service](https://ocs-docs.osisoft.com/Documentation/DataIngress/OMF_Ingress_Specification.html).
```
Comment (osimarkrogers) [inline V1/OMF/OMF_Messages_1-0.md:19]: Change "and to give it meaning" to "and give it meaning."

File: V1/OMF/OMF_Messages_1-0.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+   The value is indexed by a timestamp, and the numeric value that will be stored is a 32-bit floating point value.
+   
+2. To create the OMF type in Edge Storage, store the JSON file with the name _OmfCreateType.json_ on the local device.
+3. Run the following curl command:
+
+   ```bash
+   curl -d "@OmfCreateType.json" -H "Content-Type: application/json" -H "producertoken: x " -H "omfversion: 1.1" -H "action: create" -H "messageformat: json" -H "messagetype: type" -X POST http://localhost:5590/api/v1/tenants/default/namespaces/default/omf/
+   ```
+
+When this command completes successfully, an OMF type with the same name is created on the server. Any number of containers can be created from the type, as long as they use a timestamp as an index and have a 32-bit floating point value. The create type message needs to be sent first before container and data messages, but it does not cause an error if the same message is sent at a later time.
```
Comment (osimarkrogers) [inline V1/OMF/OMF_Messages_1-0.md:61]: Separate "first" and "before" with comma, or eliminate "first" altogether.

File: V1/OMF/OMF_Messages_1-0.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+## Container messages
+An OMF container messages uses an OMF type as a template to create a way to collect and group data events. A container message is interpreted as an SdsStream in the Sequential Data Store.
```
Comment (osimarkrogers) [inline V1/OMF/OMF_Messages_1-0.md:64]: Delete "An" and change "uses" to "use", or change "messages" to "message"

File: V1/OMF/OMF_Messages_1-0.md (hunk 2 above)
Comment (GiselleLeeOSI) [inline V1/OMF/OMF_Messages_1-0.md:61]: Not sure but does create type message need to be tagged (codeph)?

**Pull request merged**